helpers/constants.js

```diff
@@ -59,8 +59,8 @@ const FINALIZED_TASK_STATE = 2;
 const SECONDS_PER_DAY = 86400;
 
 const MINING_CYCLE_DURATION = 60 * 60 * 1; // 1 hour
-const MINING_CYCLE_TIMEOUT = 60 * 10; // Ten minutes
-const SUBMITTER_ONLY_WINDOW = 60 * 10; // Ten minutes
+const DISPUTE_DEFENCE_WINDOW = 60 * 20; // Twenty minutes
```

Comment:
This is the same variable as `CHALLENGE_RESPONSE_WINDOW_DURATION`, yes? Why different names?